### DISCUSSION TITLE
rdp gateway

### DISCUSSION THREAD

**Discussion by mhdddddd** [kimmknight/remoteapptool#144] [Q&A] [answered]
hello dude
tnx so much for your lovely remote app
i have server(win11) with ip static and when i want to connect from  outside of network it's stuck in Credentials( after inter my user and pass it's not working)
i tried by ckeck rd gateway and only use rd gateway but it's not work
.
when i use vpn server with uncheck rd gateway it's work fine
pls help me

**Comment by kimmknight** [comment]
> hello dude
> tnx so much for your lovely remote app
> i have server(win11) with ip static and when i want to connect from  outside of network it's stuck in Credentials( after inter my user and pass it's not working)
> i tried by ckeck rd gateway and only use rd gateway but it's not work
> .
> when i use vpn server with uncheck rd gateway it's work fine
> pls help me

Glad the tool has been helpful!

What are you using as an RD gateway? Microsoft's RD Gateway (Windows Server) or [rdpgw](https://github.com/bolkedebruin/rdpgw)?

**Reply by mhdddddd** [reply, marked as answer]
just normol rd for remote app from windows 11 to windows 11
just wanna connect to a app with ip static
.
update: it's be okay ifound my problem solution
tnx man pls be update in future

**Reply by MrBrianGale** [reply]
Could you tell us how you fixed it to help others? 
My guess is you had a session running on the second machine and being windows 11, it only allows 1 session at a time.
